.github/workflows/test_shark.yml

```diff
@@ -36,9 +36,9 @@ jobs:
       - name: "Checkout SHARK"
         uses: actions/checkout@v4
         with:
-          repository: "nod-ai/SHARK.git"
+          repository: "nod-ai/SHARK"
           path: SHARK
-          ref: "iree-turbine-switch"
+          ref: "diffusers-version"
```

Comment:
What is this diffusers-version? Is this your local branch?

Reply:
Yup, it's this: https://github.com/nod-ai/SHARK/pull/2157

Reply:
Ean just approved, so I can land there and change this to check out main